addons/medical_status/functions/fnc_handleKilled.sqf

```diff
@@ -41,6 +41,14 @@ if (_causeOfDeath != "#scripted") then {
         _this set [1, _killer];
         _this set [2, _instigator];
     };
+} else { // Handle setDead being skipped (forced respawn)
+
+    _unit setVariable [VAR_HEART_RATE, 0, true];
+    _unit setVariable [VAR_BLOOD_PRESS, [0, 0], true];
+
+    _unit setVariable [QEGVAR(medical,causeOfDeath), _causeOfDeath, true];
+
+    [QEGVAR(medical,death), [_unit]] call CBA_fnc_localEvent;
```

Comment:
This solution seems reasonable, but it would make sense to me to just call the setDead function here instead of duplicating the code 👍 

I see that it will result in an unnecessary call to `EFUNC(medical_engine,setStructuralDamage)`, but think that should be harmless.